.travis.yml

```diff
@@ -17,12 +18,14 @@ install:
     - git clone https://github.com/ramp-kits/iris.git
     - cd ..
 script:
-    - nosetests rampwf/tests
+    - nosetests --with-coverage rampwf/tests
     - cd kits/boston_housing
```

Comment:
don't move around with cd as coverage will be put the current dir so it will not be in a single place